homeassistant/components/onvif/parsers.py

```diff
@@ -277,6 +277,7 @@ async def async_parse_tamper_detector(uid: str, msg) -> Event:
 
 
 @PARSERS.register("tns1:Device/HardwareFailure/StorageFailure")
+@PARSERS.register("tns1:Device/Trigger/DigitalInput")
```

Comment:
Is the DigitalInput trigger really a storage failure? The logic may be the same, but we should probably create a new sensor for this instead.

Reply:
No idea. Really this camera does not have any digital input. But it looks reasonable while the SD card is missing. Feel free to create a new sensor. Thanks for looking at it.